aws/ecs_utils_for_test.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/gruntwork-io/gruntwork-cli/collections"
 	gruntworkerrors "github.com/gruntwork-io/gruntwork-cli/errors"
-	"github.com/gruntwork-io/terratest/modules/retry"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/retry"
```

Comment:
Oh this should actually be using the terratest `retry` function, which is optimized for testing (that is, it will fail the test if there is an error, and will log messages in a way that is friendly to `terratest_log_parser`).